diff --git a/dist/db/modelDB.js b/dist/db/modelDB.js
index dede2404a72441a5c5a18fcfa97e29d36bc3eb4b..737b159d138823c0881a5a5d60ac1772090265ed 100644
--- a/dist/db/modelDB.js
+++ b/dist/db/modelDB.js
@@ -26,7 +26,7 @@ import pRetry from "p-retry";
  */
 export default class ModelDB {
     constructor(db) {
-        this.remoteUrl = "https://rmbl.us";
+        this.remoteUrl = "";
         this.db = db;
     }
     static create() {
@@ -54,7 +54,7 @@ export default class ModelDB {
                 const reader = response.body.getReader();
                 let receivedLength = 0;
                 const chunks = new Uint8Array(contentLength);
-                for (;;) {
+                for (; ;) {
                     const { done, value } = yield reader.read();
                     if (done) {
                         break;
@@ -91,7 +91,7 @@ export default class ModelDB {
             }
             let tokenizer = yield this.db.getFromIndex("tokenizer", "modelID", modelID);
             if (!tokenizer) {
-                const tokenizerBytes = yield this.fetchBytes("https://huggingface.co/openai/whisper-large-v2/raw/main/tokenizer.json");
+                const tokenizerBytes = yield this.fetchBytes("/models/tokenizer.json");
                 if (tokenizerBytes.isErr) {
                     return Result.err(tokenizerBytes.error);
                 }
@@ -121,7 +121,7 @@ export default class ModelDB {
     }
     fetchRemote(model, onProgress) {
         return __awaiter(this, void 0, void 0, function* () {
-            const remoteURL = `${this.remoteUrl}/whisper-turbo/${model}-q8g16.bin`;
+            const remoteURL = `/models/${model}-q8g16.bin`;
             const fetchResult = yield this.fetchBytes(remoteURL, onProgress);
             if (fetchResult.isErr) {
                 return Result.err(fetchResult.error);
